examples/detail_info.py

```diff
@@ -0,0 +1,105 @@
+"""
+Print most info there is about the first encountered node, way, area and relation
+
+see https://docs.osmcode.org/pyosmium/master/ref_osm.html for the full documentation
+
+Note that some .pbf files may have part of their information stripped, e.g. in some downloads from https://download.geofabrik.de the user names, user IDs and changeset IDs are missing due to data protection regulations in the European Union.
+
+Author: Lukas Toggenburger
+"""
+
+import osmium
+import sys
+
+class DetailInfoHandler(osmium.SimpleHandler):
+
+    def __init__(self):
+        super(DetailInfoHandler, self).__init__()
+        self._node_showed = False
+        self._way_showed  = False
+        self._rel_showed  = False
+        self._area_showed = False
+
+    def print_obj_info(self, o):
+        print("changeset: Id of changeset where this version of the object was created (read-only): " + str(o.changeset))
```

Comment:
It would be much more useful, if it becomes visible here, what kind of type is contained in the property by using the formatting string, i.e. changeset is an int, so:

    print("changeset: %d" % o.changeset)